bitnami/kubeapps/templates/frontend/deployment.yaml

```diff
@@ -107,15 +139,28 @@ spec:
             {{- end }}
         {{- if and .Values.authProxy.enabled (not .Values.authProxy.external) }}
         - name: auth-proxy
-          image: {{ include "common.images.image" (dict "imageRoot" .Values.authProxy.image "global" .Values.global) }}
+          image: {{ include "kubeapps.authProxy.image" . }}
           imagePullPolicy: {{ .Values.authProxy.image.pullPolicy | quote }}
           {{- if .Values.authProxy.containerSecurityContext.enabled }}
           securityContext: {{- omit .Values.authProxy.containerSecurityContext "enabled" | toYaml | nindent 12 }}
           {{- end }}
+          {{- if .Values.authProxy.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.authProxy.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
```

Comment:
Looking forward to trying out the diagnosticMode options (and updating our docs to help users with the same), thanks!